pkg/kubectl/resource_printer.go

```diff
@@ -145,6 +145,9 @@ type ResourcePrinter interface {
 	// Print receives a runtime object, formats it and prints it to a writer.
 	PrintObj(runtime.Object, io.Writer) error
 	HandledResources() []string
+	//Can be used to print out warning/clarifications if needed
+	//after all objects were printed
+	FinishPrint(io.Writer) error
```

Comment:
In case of get pods without watch option set, looping on Pod list is done in RunGet function itself and PrintObj func is called to print pods one by one using printPod handler.